flink/src/main/java/org/apache/iceberg/flink/FlinkCatalog.java

```diff
@@ -359,6 +363,14 @@ public void createTable(ObjectPath tablePath, CatalogBaseTable table, boolean ig
     PartitionSpec spec = toPartitionSpec(((CatalogTable) table).getPartitionKeys(), icebergSchema);
 
     ImmutableMap.Builder<String, String> properties = ImmutableMap.builder();
+
+    // Set the equality field columns.
+    List<String> equalityFieldColumns = toEqualityColumns(table.getSchema());
+    if (!equalityFieldColumns.isEmpty()) {
+      properties.put(TableProperties.EQUALITY_FIELD_COLUMNS,
+          org.apache.commons.lang.StringUtils.join(equalityFieldColumns, ","));
```

Comment:
We try to avoid using `StringUtils` and other commons classes because we don't want to leak the dependency. Could you use Guava's `Joiner` instead?

Reply:
Sounds great.  I did not realize that there's a `Joiner` in Guava.  Thanks.